vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileViewModel.kt

```diff
@@ -149,7 +149,7 @@ class RoomProfileViewModel @AssistedInject constructor(
         combine(
                 room.flow().liveRoomMembers(roomMemberQueryParams),
                 room.flow()
-                        .liveStateEvent(EventType.STATE_ROOM_POWER_LEVELS, QueryStringValue.NoCondition)
+                        .liveStateEvent(EventType.STATE_ROOM_POWER_LEVELS, QueryStringValue.IsNotNull)
```

Comment:
@Florian14 are you ok about this update ?

Reply:
`IsEmpty`cf https://spec.matrix.org/unstable/client-server-api/#mroompower_levels